src/write_outputs/write_outputs.jl

```diff
@@ -75,11 +75,22 @@ function write_outputs(EP::Model, path::AbstractString, setup::Dict, inputs::Dic
 	elapsed_time_emissions = @elapsed write_emissions(path, inputs, setup, EP)
 	println("Time elapsed for writing emissions is")
 	println(elapsed_time_emissions)
+
+	dfVreStor = DataFrame()
+	if !isempty(inputs["VRE_STOR"])
+		dfVreStor = write_vre_stor(path, inputs, setup, EP)
+		VS_LDS = inputs["VS_LDS"]
+		VS_STOR = inputs["VS_STOR"]
+	else
+		VS_LDS = []
+		VS_STOR = []
+	end
+
 	if has_duals(EP) == 1
 		elapsed_time_reliability = @elapsed write_reliability(path, inputs, setup, EP)
 		println("Time elapsed for writing reliability is")
 		println(elapsed_time_reliability)
-		if !isempty(inputs["STOR_ALL"])
+		if !isempty(inputs["STOR_ALL"]) || !isempty(VS_STOR)
```

Comment:
Future not: we might want to rename STOR_ALL since it's now, well, not all storage.